legal-api/src/legal_api/models/consent_continuation_out.py

```diff
@@ -49,3 +52,9 @@ def get_active_cco(business_id, expiry_date, foreign_jurisdiction=None, foreign_
                 ConsentContinuationOut.foreign_jurisdiction_region == foreign_jurisdiction_region.upper())
 
         return query.all()
+
+    @staticmethod
+    def get_by_filing_id(filing_id) -> Optional[ConsentContinuationOut]:
+        """Get consent continuation out by filing_id."""
+        return db.session.query(ConsentContinuationOut). \
+            filter(ConsentContinuationOut.filing_id == filing_id).one_or_none()
```

Comment:
What happens if this DB call returns null? Won't that cause issues in report.py?

Reply:
letter of consent is available only when filing status is COMPLETED, by then this table would be populated.